app.rb

```diff
@@ -0,0 +1,25 @@
+
+require 'sinatra/base'
+require './lib/peep'
+
+class Chitter < Sinatra::Base
+  get '/' do
+    erb :homepage
+  end
+
+  get '/feed' do
+    @peeps = Peep.all
+    erb :index
+  end
+
+  get '/feed/create' do
+    erb :"feed/create"
+  end
+
+  post '/feed' do
+  Peep.create(content: params[:content])
+  redirect '/feed'
+end
```

Comment:
your indentation has got messed up here. you can run rubocop regularly to pick up things like this